pkg/addons/adminconsole/adminconsole.go

```diff
@@ -63,6 +64,28 @@ func (a *AdminConsole) HostPreflights() (*v1beta2.HostPreflightSpec, error) {
 	return a.customization.hostPreflights()
 }
 
+// addLicenseToHelmValues adds the embedded license to the helm values.
+func (a *AdminConsole) addLicenseToHelmValues() error {
+	license, err := a.customization.license()
+	if err != nil {
+		return fmt.Errorf("unable to get license: %w", err)
+	}
+	if license == nil {
+		return nil
+	}
+	raw, err := yaml.Marshal(license)
+	if err != nil {
+		return fmt.Errorf("unable to marshal license: %w", err)
+	}
+	helmValues["automation"] = map[string]interface{}{
+		"license": map[string]interface{}{
+			"slug": license.Spec.AppSlug,
+			"data": string(raw),
+		},
+	}
```

Comment:
I would _prefer_ if this was a safeadd (ie it doesn't wipe automation.somethingelse) but this works for now